zabbix/stackstate_checks/zabbix/data/conf.yaml.example

```diff
@@ -0,0 +1,9 @@
+init_config:
+    min_collection_interval: 60 # run every minute
```

Comment:
in agent v2 `min_collection_interval` has been move to the `instance`

Reply:
removed.